sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -78,41 +83,20 @@ COMMON_DEPEND=">=sys-apps/util-linux-2.30:0=[${MULTILIB_USEDEP}]
 	zstd? ( >=app-arch/zstd-1.4.0:0=[${MULTILIB_USEDEP}] )
 "
 
-# Newer linux-headers needed by ia64, bug #480218
```

Comment:
A comment about the dropping DEPEND and changes in RDEPEND would be nice. I'm not sure why are we dropping the dependency on `sys-kernel/linux-headers` for example. And why some users and groups are dropped from RDEPEND.